app/app.go

```diff
@@ -244,6 +260,18 @@ func New(dir string, options ...Option) (app *App, err error) {
 
 		cleanups = append(cleanups, func() { listener.Close(); <-proxyCh })
 
+	} else if o.Conn != nil {
+		go func() {
+			for {
+				remote := <-o.Conn.acceptCh
+				local, err := net.Dial("unix", nodeBindAddress)
+				if err != nil {
+					continue
```

Comment:
Why are errors ignore here after accepting an inbound request?

Reply:
Just wondering also what we do with the accepted connection?

Reply:
The remote accepted connection? We just copy any data back and forth between the accepted connection and dqlite's bind address. Did you mean something specific?

Reply:
Yeah I don't see any problem with closing it.